desktop.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=Robo 3T
+GenericName=MongoDB GUI
+Comment=Free lightweight GUI for MongoDB enthusiasts
+Exec=robo3t
+Icon=org.robomongo.Robo3T
+Type=Application
+Categories=Development
```

Comment:
It would be nice if you can add more categories https://standards.freedesktop.org/desktop-entry-spec/latest/ar01s06.html and Keywords.